js/site.js

```diff
@@ -20,12 +40,17 @@ class Site{
     getScore(){
         this.score = this.scoreFunction();
         return this.score;
-    }
+    };
 
-    getTrackers(){ return this.trackers };
+    _checkGlobalWhitelist(domain){
```

Comment:
I think it makes no sense to have function that checks the global setting and returns it, which you use to skip your internal method of setting the internal value.

This should be called something like setWhitelistStatusFromGlobal() which internally calls setWhitelisted() with the value.